lib/worker.py

```diff
@@ -106,7 +106,7 @@ def reproduce_task_groups(task_groups):
     for i, task_id in enumerate(reproduce):
         for key, task_group in task_groups.items():
             if task_id in task_group['task_ids']:
-                found_keys.append(key)
+                found_keys.append(key.rstrip("_fragile"))
```

Comment:
Nit: Follow quotes style of the surrounding code, here they are single quotes.

Reply:
Sorry. This code style is the opposite for me. Fixed it.